src/MMALSharp.Processing/Handlers/FrameBufferCaptureHandler.cs

```diff
@@ -45,6 +44,19 @@ public FrameBufferCaptureHandler()
             : base()
         { }
 
+        /// <summary>
+        /// Creates a new <see cref="FrameBufferCaptureHandler"/> configured for motion detection using a raw video stream
+        /// where MMALStandalone.Instance is used (such as processing a pre-recorded file) rather than camera-based processing.
+        /// </summary>
+        /// <param name="motionConfig">The motion configuration.</param>
+        /// <param name="onDetect">A callback for when motion is detected.</param>
+        public FrameBufferCaptureHandler(MotionConfig motionConfig, Action onDetect)
```

Comment:
I'm assuming this additional constructor is used to replicate what `MMALCamera.WithMotionDetection()` does?

Reply:
By funny coincidence I was just looking at that myself, and I don't think it is needed. There was a point where I was trying to avoid modifying `MotionConfig` and I think it's left over from that. I will straighten it out.

Reply:
Actually that constructor was useful when doing nothing but visualizing (the US/UK s/z thing keeps messing with me!) the motion detection analysis. Used in a simple configuration like this:

```csharp
using (var shell = new ExternalProcessCaptureHandler(raw_to_mjpeg_stream))
using (var motion = new FrameBufferCaptureHandler(motionConfig, null))
using (var resizer = new MMALIspComponent())
{
    motionAlgorithm.EnableAnalysis(shell);

    resizer.ConfigureOutputPort<VideoPort>(0, new MMALPortConfig(MMALEncoding.RGB24, MMALEncoding.RGB24, width: 640, height: 480), motion);
    cam.Camera.VideoPort.ConnectTo(resizer);

    await cameraWarmupDelay(cam);
    var timeout = new CancellationTokenSource(TimeSpan.FromSeconds(seconds));
    await Task.WhenAll(new Task[]{
            shell.ProcessExternalAsync(timeout.Token),
            cam.ProcessAsync(cam.Camera.VideoPort, timeout.Token),
        }).ConfigureAwait(false);
}

```

Reply:
Oh I see I do explain one usage in the constructor summary, although it could be better, it does now also work with the camera (the example above is for streaming). I'll update the summary.

> Creates a new <see cref="FrameBufferCaptureHandler"/> configured for motion detection using a raw video stream where MMALStandalone.Instance is used (such as processing a pre-recorded file) rather than camera-based processing.